config/schemas/config.yaml

```diff
@@ -126,6 +363,658 @@ properties:
+        type: array
+      additionalIngressPolicies:
+        type: array
+      alertmanager:
+        additionalProperties: false
+        properties:
+          alertReceivers:
+            $ref: '#/$defs/listen'
+        type: object
+      allowedNameSpaces:
+        type: array
+      certManager:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          letsencrypt:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+            type: object
+          namespaces:
+            items:
+              default: dex
+              type: string
+            type: array
+          dns01:
+            type: object
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+        type: object
+      coredns:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          externalDns:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+            type: object
+          serviceIp:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+            type: object
+        type: object
+      defaultDeny:
+        default: false
+        type: boolean
+      dex:
+        additionalProperties: false
+        properties:
+          connectors:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      dnsAutoscaler:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      enableAlerting:
+        default: true
+        type: boolean
+      falco:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          plugins:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+        type: object
+      fluentd:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      gatekeeper:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      global:
```

Comment:
Seem to be missing some properties here:
```console
wc-config.yaml: networkPolicies.kubeSystem: Additional property openstack is not allowed
wc-config.yaml: networkPolicies.alertmanager: Additional property enabled is not allowed
wc-config.yaml: networkPolicies.global: Additional property wcApiserver is not allowed
wc-config.yaml: networkPolicies.global: Additional property wcNodes is not allowed
```

Reply:
How's https://github.com/elastisys/compliantkubernetes-apps/pull/2065/commits/82441f88d61b3296924e9a42a1548b161669cc37 ?

Reply:
Now I get this, as some of our netpols have an `enabled` property (not all of them it seems, as we are using different netpol charts for some apps):
```
wc-config.yaml: networkPolicies.kubeSystem.upcloud: Additional property enabled is not allowed
```

I also noticed some missing properties for certManager:
```bash
sc-config.yaml: certmanager.tolerations: Invalid type. Expected: array, given: object
sc-config.yaml: certmanager.webhook.tolerations: Invalid type. Expected: array, given: object
sc-config.yaml: certmanager.cainjector: Additional property extraArgs is not allowed
sc-config.yaml: certmanager.cainjector.tolerations: Invalid type. Expected: array, given: object
```

The `tolerations` ones should be updated to the correct type (in this case changed to array) in the [config](https://github.com/elastisys/compliantkubernetes-apps/blob/v0.37.0/config/config/common-config.yaml#L711). Since we noticed other cases where the config should be updated, I was wondering if you are planning to update such issues in these PRs or would that be a follow up task?

Reply:
Probably worth prioritizing CI blockers.

The end goal is that schema vs config disagreements can be resolved by a PR from anyone. It doesn't have to be perfect at first.